pkg/workloads/console/controller.go

```
@@ -137,6 +137,10 @@ func (r *reconciler) Reconcile() (res k8rec.Result, err error) {
 		return res, err
 	}
 
+	// Set the template as owner of the console
+	// This means the console will be deleted if the template is deleted
+	r.SetConsoleOwner(consoleTemplate)
```

error result isn't being checked, so this can silently fail 😢